lib/vagrant-libvirt/action/create_network_interfaces.rb

```diff
@@ -213,9 +213,6 @@ def call(env)
                 network[:type] = :dhcp
               end
 
-              # do not run configure_networks for tcp tunnel interfaces
-              next if options.fetch(:tunnel_type, nil)
```

Comment:
@axmetishe Do you mean this line in your last sentence? I can return a value for auto_config to false.

Reply:
No, I talked about  `next if options[:auto_config] === false` part.
Part regarding tunnels auto config `next if options.fetch(:tunnel_type, nil)` is crucial for project you are working on(if I'm not missed something=)

Reply:
@h0st sorry for late response